marlowe-dashboard-client/plutus-pab.yaml

```diff
@@ -14,7 +14,6 @@ pabWebserverConfig:
 
 walletServerConfig:
   tag: LocalWalletConfig
-  walletSettings:
     baseUrl: http://localhost:9081
```

Comment:
Is there an indentation mistake here?

Reply:
No, the configuration is nested one layer deeper now.